src/RageSoundReader_Extend.cpp

```diff
@@ -155,7 +155,7 @@ bool RageSoundReader_Extend::SetProperty( const RString &sProperty, float fValue
 {
 	if( sProperty == "StartSecond" )
 	{
-		m_iStartFrames = std::lrint( fValue * this->GetSampleRate() );
+		m_iStartFrames = static_cast<int64_t>((fValue * this->GetSampleRate()) + 0.5);

```

Comment:
All these frames are currently ints (https://github.com/itgmania/itgmania/blob/04647648765b082414993e6f2ce420b2d950840b/src/RageSoundReader_Extend.h#L28). We can keep the data types as ints for now instead of int64_t